frontend/src/App.jsx

```diff
@@ -12,11 +12,25 @@ function App() {
   const { defaultAlgorithm, darkAlgorithm } = theme;
   const { sessionDetails } = useSessionStore();
   const { alertDetails } = useAlertStore();
+  console.log(alertDetails);
```

Comment:
We sure about needing this console.log?

Reply:
No we don't need, I removed and pushed the changes.